Results discovery write up (#554)
NHSDigital/prevention-services-design-history: app/managing-my-health/2026/07/helping-users-do-more-with-results/index.md

- timely access to their results, without having to chase
- plain language they can understand
- clear next steps – what to do, or reassurance that nothing is needed
- a way to reach a person when they need one

However, the experience sometimes does not meet these needs. Many results from prevention services appear in the test results feed in the NHS App. This often surfaces content straight from labs, in unclear clinical language, before a healthcare professional has explained what it means. When looking at their results, one user told us:

> I know what HPV is, but so it says positive for HPV. It said grade 7. I did not know what that meant. So, I probably Googled it… It says cervical cytology report, high grade... Squamous, I can’t pronounce it. So, then you think high grade, that probably in this situation is bad

Because the feed pulls results automatically from the GP health record, prevention teams don’t control what appears there. It isn’t how they want results to reach people.

Teams continue to use messages and letters to provide context and reassurance with results. But those sit apart from the feed. So users get an inconsistent experience across multiple channels.

> Managing my health sits between prevention services and the NHS App. As the app shifts from a transactional focus to a companion that follows up over time, results are a place to start – they’re personal and a potential first step towards more connected journeys.

Reviewer: Add link here?

```suggestion
Managing my health sits between prevention services and the NHS App. As the app shifts [from a transactional focus to a companion](https://digital.nhs.uk/blog/transformation-blog/2026/the-nhs-app-from-digital-front-door-to-lifelong-companion) that follows up over time, results are a place to start – they’re personal and a potential first step towards more connected journeys.
```

We want to give people a clear results loop:
- they’re notified when there’s something to know
- they see a record that makes sense

> - they’re offered a clear next step, if needed

Reviewer: Update so that the lead-in and the bullets work together as a sentence?

```suggestion
We want people to:

- be notified when there’s something to know
- see a record that makes sense
- have clear next steps, if needed
```

Currently, that loop breaks in different ways depending on how the result reaches them.

We ran a discovery to see if there’s an opportunity to show results in a consistent way, with context and clear next steps, and to create the foundations for those journeys. We focused on the two channels we can most influence, and that point to the long‑term direction for services – the app’s results feed and messages.

## How users get results

There are 3 main ways users get results in the NHS App:
- the test results feed
- documents
- in‑app messages

![NHS App home screen with lines pointing from ‘Results can appear in:’ to the Test results and Documents menu items and the Messages tab.](places-for-results.png "Results can reach users in 3 places in the NHS App – the test results feed, documents, and messages.")

Each has limitations, and they’re not mutually exclusive. For most screening results, users get both a message (in the app, by SMS, or by letter) and the same result in the test results feed.

### Test results feed

> Results move from the lab to the GP record to the app, often before a clinician has added any context. Roughly speaking, this is the sequence by which results arrive in the NHS App’s test results feed:

Reviewer: Bit clearer if the line before the list stands as its own paragraph maybe?

```suggestion
Results move from the lab to the GP record to the app, often before a clinician has added any context.

Roughly speaking, this is the sequence by which results arrive in the NHS App’s test results feed:
```

> 1. Result produced – a sample is tested. The lab’s Laboratory Information Management System (LIMS) records the result, and a clinician validates and authorises it

Reviewer: Non-breaking spaces are cool and all, but this is quite a long phrase not to break across 2 lines, especially on mobile.

```suggestion
1. Result produced – a sample is tested. The lab’s Laboratory Information Management System (LIMS) records the result, and a clinician validates and authorises it
```

### Documents

The flow for results given to users via documents is roughly the same, although documents won’t have started in a LIMS. Documents are usually correspondence, though results can be included in them.

Documents reach the GP health record via clinical document transfer routes – for example GP Connect: Send Document, which also uses MESH to carry the file. They arrive in a GP practice’s document management system, where staff review and then file them. Filing pushes the documents into the GP health record.

The NHS App reads the record (via IM1 or GP Connect) and lists them under Documents. Users open or download them – viewed as they were sent, no transformation.

### Messages

Teams can send users messages that include results. NHS Notify can send these as in‑app messages, SMS, email, or letter. The in‑app and SMS versions are formatted text and can contain links.

## How the way users get results affects their experience

How a result reaches someone shapes how they experience it. 3 things matter most:
- the words used to convey it
- the speed and discretion with which it arrives
- the noise and structure it sits within

### Content design

> The test results feed surfaces results directly from the lab. It’s possible for the lab or GP practice to add content before the result reaches the user, but in most cases users get results as they come from the lab. The results contain medical terms and are not designed to give reassurance or clarity. We have seen examples where users found the results confusing and distressing. Faced with a result they can’t understand, people turn to Google and AI tools to work out what it means.

Reviewer: Can we add an example screenshot here?

NHS Notify gives teams much greater control over what users are sent and when. The in‑app and SMS messages are limited to text and links, but a well‑written message – especially where there’s no need for immediate action – gives users the information they need to feel reassured.

We didn’t spend much time in discovery looking at documents. Their content relies entirely on the service upstream. Our focus was on teams using the app to convey messages to users.

### Speed and discretion

For prevention, the way a result arrives matters.

Of the 3 routes, the test results feed is likely to reach a user first. A result can flow from the lab into the feed automatically. National guidance sets release timing for the most sensitive results – some are held back for weeks, and some, like genetic testing, are never released this way. But the feed relies on that timing being set correctly. When it isn’t, a result can appear before a clinician has reviewed it.

> The test results feed can’t notify users when a result is ready. It can’t tell whether they’ve seen it either. A message does both. It sends a notification, and because it knows whether it’s been read, it can fall back to SMS or another channel if the user misses it.

Reviewer: Avoid negative contractions? (a style guide point - but also makes the point a bit more powerfully?)

```suggestion
The test results feed does not notify users when a result is ready. It does not show whether they’ve seen it either. A message does both. It sends a notification, and because it knows whether it’s been read, it can fall back to SMS or another channel if the user misses it.
```

The principle within the NHS has generally been that a person should not receive a potentially concerning result without the support of a healthcare professional. That safeguard relies on a clinician reviewing a result before the patient sees it. In the feed route, that review is meant to happen when the result is filed into the GP record. But the result can surface in the feed without being reviewed. A distressing result can reach someone with no explanation and no next step.

The cervical screening team sends its messages as quickly as it can, covering all outcome types, so whatever the result, the user gets plain language and a clear sense of what it means. Their work shows it’s possible to give people a range of results and, with good content, still offer the reassurance they need.

### Noise and structure

The channel a result arrives in also shapes the experience. Both the test results feed and the message inbox can fill with noise, with important information shown in the same hierarchy as everything else.

> The feed has no read or unread status either. Nothing shows a user which results they’ve already looked at. Messages mark read and unread, which helps people keep track.

Reviewer: This point is already made above. De-dupe?

Author: Done
